packages/main/src/themes/TabContainer.css

```diff
@@ -98,8 +98,12 @@
 }
 
 .ui5-tc__headerItem--withIcon:focus .ui5-tc-headerItemIcon-outer,
+.ui5-tc__headerItem--textOnly:not(.ui5-tc__headerItem--inline):focus .ui5-tc__headerItemText {
```

Comment:
I would prefer if all class names are written with __ or _ rather than --, but as much as I can see some of them are using -- before this change as well.

Reply:
Thanks, but this is Adrian's css, actually he is the only one followed correctly BEM, I will leave it to be complaint with the rest of the file